core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -990,7 +991,12 @@ portfolio.failed.cantUnfail=This trade cannot be moved back to open trades at th
   Try again after completion of trade(s) {0}
 portfolio.failed.depositTxNull=The trade cannot be reverted to a open trade. Deposit transaction is null.
 portfolio.failed.delayedPayoutTxNull=The trade cannot be reverted to a open trade. Delayed payout transaction is null.
+portfolio.failed.penalty.msg=This will charge the buyer/seller as {4} the trade fee of {0} XMR and return the remaining trade funds of {1} XMR to their wallet. Are you sure you want to send?\n\n\
+  Other Info:\n\
+  Transaction Fee: {2} XMR\n\
+  Reserve Tx Hash: {3}\n\
 
+portfolio.failed.error.msg=Trade record does not exist; contact administrator.
```

Comment:
`portfolio.failed.error.msg` is too generic. `portfolio.failed.noSignedOffer` ?

Reply:
Changed accordingly in the `displayStrings.properties` file.

![Screenshot from 2023-02-28 16-06-35](https://user-images.githubusercontent.com/20237127/221894122-6486dc1b-1f43-421a-a518-1532955af68e.png)